tokens/SOE.json

```diff
@@ -0,0 +1,2 @@
+[
```

Comment:
what doea it mean?

Reply:
I guess you need to add the token information

Reply:
we do not need to do that, and this empty file cannot pass the test ;D
Whats' more, no worry about the worng name of the file, we would check it

Reply:
so please close it